OgcApi.Net.Features.PostGis/PostGisProvider.cs

```diff
@@ -38,5 +39,24 @@ protected override Geometry ReadGeometry(DbDataReader dataReader, int ordinal, S
 
             return geometryReader.Read((byte[])dataReader.GetValue(ordinal));
         }
+        public override ICollectionSourceOptions DeserializeCollectionSourceOptions(string json, JsonSerializerOptions options)
+        {
+            return JsonSerializer.Deserialize<SqlCollectionSourceOptions>(json, options);
+        }
+
+        public override void SetCollectionOptions(ICollectionsOptions options)
+        {
+            if (options == null)
+                throw new ArgumentNullException(nameof(options));
+            CollectionsOptionsValidator.Validate(options as CollectionsOptions);
+            if (options is CollectionsOptions collectionOptions
+                && collectionOptions?.Items != null
+                && collectionOptions.Items.Any(i => i.Features.Storage.Type == SourceType))
+            {
+                CollectionsOptions = new CollectionsOptions();
```

Comment:
Declare new variable with the CollectionsOptions type to not cast it every time.